src/components/ReportActionItem/IOUPreview.js

```diff
@@ -124,61 +131,71 @@ const IOUPreview = (props) => {
                 {reportIsLoading
                     ? <ActivityIndicator style={styles.iouPreviewBoxLoading} color={themeColors.text} />
                     : (
-                        <View>
-                            <View style={styles.flexRow}>
-                                <View style={styles.flex1}>
-                                    <View style={styles.flexRow}>
-                                        <Text style={styles.h1}>
-                                            {cachedTotal}
-                                        </Text>
-                                        {!props.iouReport.hasOutstandingIOU && (
-                                            <View style={styles.iouPreviewBoxCheckmark}>
-                                                <Icon src={Expensicons.Checkmark} fill={themeColors.iconSuccessFill} />
-                                            </View>
-                                        )}
+                        <OfflineWithFeedback
+                            pendingAction={CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD}
+                            errors={props.walletTerms.errors}
+                            onClose={() => {
+                                PaymentMethods.clearWalletTermsError();
+                                Report.clearIOUError(props.chatReportID);
+                            }}
+                            errorRowStyles={[styles.mbn1]}
+                        >
+                            <View>
+                                <View style={styles.flexRow}>
+                                    <View style={styles.flex1}>
+                                        <View style={styles.flexRow}>
```

Comment:
NAB since I know these aren't related to your changes but that sure is a whole lotta nested Views, I don't think those are all necessary